srcpkgs/runit-void/files/09-apparmor.sh

```diff
@@ -19,6 +19,9 @@ if [ -n "$APPARMOR" ]; then
 
 	if [ -d /etc/apparmor.d -a -x /usr/bin/apparmor_parser ]; then
 		for profile in /etc/apparmor.d/*; do
+		case "$profile" in
+			*.new-[0-9]*_[0-9]*) continue ;;
```

Comment:
Does it need to be "robust", I even think `*.new-*` would be fine.
Wouldn't your pattern already break with basically all packages that drop config files into `/etc/apparmor.d` because they have dots in their version?